src/components/Search/SearchBar.js

```diff
@@ -68,7 +72,7 @@ function SearchBar(props) {
             e.target.keyCode === 13 || e.key === "Enter";
     };
     return (
-        <MuiThemeProvider theme={CONSTANTS.THEME()}>
+        <>
```

Comment:
Empty `<>` ? What does that do ?

Reply:
Short syntax for `React.Fragment`: https://reactjs.org/docs/fragments.html#short-syntax